### PR TITLE
Tecg 382 user timezone and others issues

### PR DESCRIPTION
**Pull request by brunoliberal**

### CODE REVIEW

File: src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java [hunk 1]
```diff
@@ -166,18 +179,23 @@ public void sendEmailToEndorseds() {
     try {
       List<TechGalleryUser> usersList = techGalleryUserDao.findAll();
       for (TechGalleryUser techGalleryUser : usersList) {
-        List<Endorsement> endorsementsList = endorsementDao.findAllEndorsementsStartingFrom(techGalleryUser,
-            lastExecutedCronJob);
+        List<Endorsement> endorsementsList =
+            endorsementDao.findAllEndorsementsStartingFrom(techGalleryUser, lastExecutedCronJob);
         if (endorsementsList != null) {
-          TechGalleryActivitiesTO activities = new TechGalleryActivitiesTO(Constants.APP_NAME, null, new ArrayList<TechnologyActivitiesTO>());
+          TechGalleryActivitiesTO activities =
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java]: @brunoliberal I think would be great if email TOs name be related to email. I mean something like TechGalleryActivitiesEmailTemplateTO. Also put it in an email TO package.

File: src/main/java/com/ciandt/techgallery/utils/timezone/TimezoneManager.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package com.ciandt.techgallery.utils.timezone;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class TimezoneManager {
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/utils/timezone/TimezoneManager.java]: @brunoliberal Please create a javadoc to this class.

File: src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java (hunk 1 above)
Comment (brunoliberal) [inline src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java]: Done

File: src/main/java/com/ciandt/techgallery/utils/timezone/TimezoneManager.java (hunk 2 above)
Comment (brunoliberal) [inline src/main/java/com/ciandt/techgallery/utils/timezone/TimezoneManager.java]: Done.

**Pull request merged**